docs/status.md

```diff
@@ -159,3 +159,18 @@ one of the following tracking labels.
 | uniform_quantize         | yes           | revisit      | infeasible     | yes             | no          |
 | while                    | yes           | revisit      | yes            | revisit         | yes         |
 | xor                      | yes           | yes          | yes            | yes             | yes         |
+
+## Type inference for quantized operations
+
+The `Type Inference` column from the table above is intended to focus on
+non-quantized operations. For the majority of the quantized operations, it is
+not feasible to infer the result type because the quantization parameters of
+the result types may vary from those of the operands, with the exception of
+few cases that are listed below: `all_gather`,
+`all_to_all`, `case`, `collective_permute`,
```

Comment:
why `dot_general` because we cannot infer the storage type of result 

```
(C15) storage_type(lhs) = storage_type(rhs).
(C16) expressed_type(lhs) = expressed_type(rhs) = expressed_type(result).
```

Reply:
Similarly we cannot infer the quantization parameter for result of a `uniform_quantized` operation.

Reply:
yup..`storage_type ` can't be inferred. 

`uniform_quantized`.. forgot to remove. thanks for catching those.